tdrs-frontend/src/components/FileUpload/FileUpload.jsx

```diff
@@ -167,6 +193,17 @@ function FileUpload({ section, setLocalAlertState }) {
         aria-hidden="false"
         data-errormessage={INVALID_FILE_ERROR}
       />
+      <div style={{ marginTop: '25px' }}>
+        {hasFile ? (
+          <Button
+            className="tanf-file-download-btn"
+            type="button"
+            onClick={downloadFile}
+          >
+            Download
```

Comment:
@riatzukiza would it be a relatively low lift to append  "Section" + sectionNumber to this so that we can match the figma mockups (e.g. Download Section 1, etc.)? [ref](https://github.com/raft-tech/TANF-app/pull/859#issuecomment-874325081)

Reply:
Can do